package.json

```diff
@@ -30,7 +30,7 @@
     "react-native-orientation": "enahum/react-native-orientation.git",
     "react-native-push-notification": "3.0.0",
     "react-native-search-bar": "enahum/react-native-search-bar.git",
-    "react-native-sentry": "getsentry/react-native-sentry.git",
+    "react-native-sentry": "^0.10.0",
```

Comment:
@jarredwitt can we use `yarn add -E react-native-sentry` just to be sure we are using the exact version please